source/S_Scalar_Line_Integral.ptx

```diff
@@ -0,0 +1,496 @@
+    <me> \sum_{i=0}^{k-1} f(r_i^{*}) \Vert{r_{i+1}-r_{i}}\Vert</me>
+    where <m>f(r_i^{*})</m> is the output of <m>f</m> for some <m>r_i^{*} \in [r_i,r_{i+1}]</m>. As this sum is made with more pieces and all of the lengths of the pieces goes to zero (i.e. <m>\Vert{r_{i+1}-r_{i}}\Vert \rightarrow 0</m>), we would expect that the sum will approach the actual acculumation of <m>f</m> over <m>C</m>. Notice that it won't matter how we select the point <m>r_i</m> that used in each piece to evaluate the output of <m>f</m> since evaluating the limit as the pieces gets smaller will ensure that the output value will be within a shrinking error from the average value on each piece.
+    Evaluating the limit as the size of all of the pieces goes to zero will transform our Riemann sum into an integral that will measure the accumulation of the output of <m>f</m> over <m>C</m>.
+  </p>
+  <definition xml:id="ScalarLineIntegral_def">
+    <title>The Line Integral of a Scalar Function</title>
+    <p>
+      Let <m>f</m> be a contiunuous function of <m>x</m>, <m>y</m>, and <m>z</m> for some open set around <m>C</m>, a curve from a point <m>P</m> to a point <m>Q</m> in space. The line integral of <m>f</m> over <m>C</m> is defined as
+      <me>\int_C f ds =\lim_{k\rightarrow \infty} \sum_{i=0}^{k-1} f(r_i^{*}) \Vert{r_{i+1}-r_{i}}\Vert</me>
+      where <m>r_i</m> are points such that <m>P=r_0, r_1, ..., r_k-1, r_k=Q</m> and as <m>k\rightarrow \infty</m> the distance between <m>r_{i-1}</m> and <m>r_i</m> goes to zero.
+    </p>
+    <p>The integral 
+      <me>\int_C f ds </me>
+      is called the scalar line integral of <m>f</m> over the curve <m>C</m>.
+    </p>
+  </definition>
+  <p>
+    Do we need to talk more about why this limit of ths sum will converge?
+  </p>
+  <p>
+    The notation for a scalar line integral (<m>\int_C f ds </m>) may not immediately make sense. As with the other types of integration we have done (double integrals, line integrals of vector fields, single variable integration, etc.), the subscript of the integral symbol denotes the region of integration. In the case of a scalar line integral, the region of integration is a collection of points given by a curve in space. The function we are integrating is <m>f</m>, a scalar valued funciton (of multiple variables). The differential <m>ds</m> probably looks suspicious to you. If you remember from Section <xref ref="S-9-8-Arc-Length-Curvature" />, <m>s</m> is the arc length of a curve in space. So the differential <m>ds</m> in the scalar line integral notation means that we are adding up the output of <m>f</m> over steps in arc length. This should make sense in terms of how we set up our Riemann sums. We did not set up the pieces of our curve as steps in <m>x</m>, <m>y</m>, or <m>z</m>, but rather as steps in arc length (estimated by <m>\Vert{r_{i+1}-r_{i}}\Vert</m>).
+  </p>
+
+  <activity xml:id="Act_ScalarLI1">
```

Comment:
I feel like the structure on this could be a bit better (especially since the `introduction` probably isn't ideal), but that can be experimented with a bit. I might make `introduction` a general description of the point of the `activity`, rather than having part of the prompt in it (especially since it doesn't apply to the first task). Note that `task` can be structured with more `task` inside it, so rather than using an `ol` inside each of these tasks, you could use an `introduction` that says what the students are to do and then each `task` inside the `task` would be a single curve or line integral. Frankly, for the sake of space, I might put the list of functions in the `introduction` to the activity and then change the statement of most of the tasks to just refer to the list of functions that comes before.

Reply:
I think in the latest version I removed all of the png images. Those were a first pass drawing in MS Paint but I have since added svg and pdf versions of everything in my commits.
I will make the structure changes to that activity in my next commit.

Reply:
Sounds good. Before getting to merging the PR, try to remove the PNG images so that we have a big cleaner situation for what's in the official repo.